### PR TITLE
reside-46: blog post on deferring errors

### PR DESCRIPTION
**Pull request by richfitz**

### CODE REVIEW

File: content/blog/deferring-errors.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+                 paste0("  - ", errors, collapse = "\n")),
+         call. = call)
+  }
+
+  value
+}
+```
+
+So when an error of `deferrable_error` occurs, `withCallingHandlers` invokes the handler which collects the error up for later, then involkes the restart (registered by the error itself) which does nothing but _somehow_ signals that we can continue.
+
```
Comment (weshinsley) [inline content/blog/deferring-errors.md:122]: Hurrah, a typo! involkes

**Review by weshinsley — CHANGES_REQUESTED** [review]
One typo! But this is really cool. A couple of thoughts - not sure if they are worth including:

1. This is especially useful if there is a large upfront delay (eg, read large files), before it gets to the code that fails. The stochastic import is a good example where I'm sure I had times of iteratively fixing the only reported error, with a file-read delay each time...

2. And I suppose at times you might be unlucky and have an error avalanche where all the later errors go away if you fix the first one... So there's a small art of choosing when to defer and when to bail out.

File: content/blog/deferring-errors.md (hunk 1 above)
Comment (richfitz) [reply content/blog/deferring-errors.md:122]: fixed, along with some other typos :see_no_evil:

**Review by weshinsley — APPROVED** [review]
Looks great to me!

**Review by r-ash — APPROVED** [review]
Really interesting stuff and great write up. 

Should we link the blog post from the repo too?

**Comment by richfitz** [comment]
Good idea on the linking @r-ash

**Pull request merged**